### PR TITLE
Partially Update 

### PR DESCRIPTION
**Pull request by MadsDueKristensen**
reslove #48 

Comes from #47

### CODE REVIEW

**Comment by NJohansen** [comment]
We need to only include the partially update code in this PR because right now this PR consists of two PR's.

**Comment by kvartborg** [comment]
Will be automatically fixed if the #47 is merged first.

File: TaskHouseApi/Persistence/Repositories/Repository.cs [hunk 1]
```diff
@@ -50,8 +52,55 @@ public void Delete(int id)
...
+
+                /// Gets the property value
+                var propertyValue = property.GetValue(baseModel);
+
+                /// If the value is null
+                if (propertyValue == null)
+                {
+                    /// ??????????
+                    /// SHOULD THE property be set to null ?
+                    /// ??????????
```
Comment (kvartborg) [inline TaskHouseApi/Persistence/Repositories/Repository.cs]: Do we have any properties which we need to set to null?

File: TaskHouseApi/Persistence/Repositories/UserRepository.cs [hunk 2]
```diff
@@ -29,7 +29,8 @@ public bool DeleteRefrechToken(RefreshToken refreshToken)
 
         public bool isInDatabase(int Id)
         {
-            return !postgresContext.Users.Any(o => o.Id == Id);
+            var r = postgresContext.Users.Any(o => o.Id == Id);
```
Comment (kvartborg) [inline TaskHouseApi/Persistence/Repositories/UserRepository.cs]: No need to assign it to a var just return it

File: TaskHouseUnitTests/FakeRepository.cs [hunk 3]
```diff
@@ -47,5 +49,52 @@ public void Update(T baseModel)
...
+
+                /// Gets the property value
+                var propertyValue = property.GetValue(baseModel);
+
+                /// If the value is null
+                if (propertyValue == null)
+                {
+                    /// ??????????
+                    /// SHOULD THE property be set to null ?
+                    /// ??????????
```
Comment (kvartborg) [inline TaskHouseUnitTests/FakeRepository.cs]: Remove.

File: TaskHouseApi/Controllers/WorkersController.cs [hunk 4]
```diff
@@ -49,6 +50,7 @@ public IActionResult Create([FromBody]Worker worker)
             {
                 return BadRequest(new { error = "CreateWorker: worker is null" }); // 400 Bad request
             }
+            PropertyInfo[] propertyInfo = worker.GetType().GetProperties();
```
Comment (NJohansen) [inline TaskHouseApi/Controllers/WorkersController.cs]: Do you use propertyInfo in this call?

File: TaskHouseApi/Controllers/WorkersController.cs [hunk 5]
```diff
@@ -69,19 +71,21 @@ public IActionResult Create([FromBody]Worker worker)
...
 
-            if (!unitOfWork.Workers.isInDatabase(w.Id))
+            if (!unitOfWork.Workers.isInDatabase(id))
             {
                 return NotFound();
             }
 
-            unitOfWork.Workers.Update(w);
+            w.Id = id;
+            //unitOfWork.Workers.Update(w);
```
Comment (NJohansen) [inline TaskHouseApi/Controllers/WorkersController.cs]: Comment

**Pull request merged**